Docs/sphinx/Equations.rst

```diff
@@ -22,32 +22,30 @@ PeleC advances the following set of fully compressible equations for the conserv
  
   \frac{\partial \rho}{\partial t} &=& - \nabla \cdot (\rho \mathbf{u}) + S_{{\rm ext},\rho},
 
-  \frac{\partial (\rho \mathbf{u})}{\partial t} &=& - \nabla \cdot (\rho \mathbf{u} \mathbf{u} -p{\mathcal I} + \Pi) - \nabla p +\rho \mathbf{g} + \mathbf{S}_{{\rm ext},\rho\mathbf{u}},
+  \frac{\partial (\rho \mathbf{u})}{\partial t} &=& - \nabla \cdot (\rho \mathbf{u} \mathbf{u} + \Pi) - \nabla p +\rho \mathbf{g} + \mathbf{S}_{{\rm ext},\rho\mathbf{u}},
 
-  \frac{\partial (\rho E)}{\partial t} &=& - \nabla \cdot (\rho \mathbf{u} E + p \mathbf{u}) + \rho \mathbf{u} \cdot \mathbf{g} + \nabla\cdot \boldsymbol{\mathcal{Q}}+ S_{{\rm ext},\rho E},
+  \frac{\partial (\rho E)}{\partial t} &=& - \nabla \cdot (\rho \mathbf{u} E + p \mathbf{u}) - \nabla \cdot (\Pi \mathbf{u}) + \rho \mathbf{(u+u_{d})} \cdot \mathbf{g} + \nabla\cdot \boldsymbol{\mathcal{Q}}+ S_{{\rm ext},\rho E},
```

Comment:
A few things here: 
- The term involving the diffusion velocity should have summation over species because the diffusion velocity is defined individually for each species. I would also prefer a capital V to denote diffusion velocity for consistency with common combustion texts such as Law's and Poinsot's
- The viscous stress term should look like `\nabla \cdot (\mathbf{u} \cdot \Pi)` - there need to be two contractions to reduce the second order tensor \Pi to a scalar term.